### PR TITLE
contact us enabled

### PR DESCRIPTION
**Pull request by alias01**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -3,6 +3,10 @@ import Buefy from 'buefy';
 import App from './App.vue';
 import 'buefy/dist/buefy.css';
 import router from './router'
+import VeuResource from 'vue-resource';
```
Comment (arrogant-ant) [inline src/main.js:6]: please remove additional import

File: src/pages/home/components/Contact.vue [hunk 2]
```diff
@@ -18,28 +18,69 @@
...
-						.control.level
-							.select.level-left
-								select(id="options" name="Authority type")
-									option Parking Operator
-									option Parking User
-									option Others
+						input(id="cityc" v-model="City" class="input" type="text" name="city" placeholder="Bangalore")
+						label.label.has-text-black.has-text-left Contact No.
+						input(id="mobilec" v-model="Mobile" class="input" type="tel" name="mobile" placeholder="+91 ")
+						//- label.label.has-text-black.has-text-left Authority type
```
Comment (arrogant-ant) [inline src/pages/home/components/Contact.vue:30]: please remove unwanted code

**Review by arrogant-ant — CHANGES_REQUESTED** [review]

File: src/main.js (hunk 1 above)
Comment (alias01) [reply src/main.js]: removed extra import

File: src/pages/home/components/Contact.vue (hunk 2 above)
Comment (alias01) [reply src/pages/home/components/Contact.vue]: removed unwanted code

**Review by arrogant-ant — APPROVED** [review]

**Pull request merged**